schemas/com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Common Context Schema for a media player event",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "currentTime": {
+      "type": "number",
+      "description": "The current playback time",
+      "minimum": 0,
+      "maximum": 9007199254740991
+    },
+    "duration": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "A double-precision floating-point value indicating the duration of the media in seconds",
+      "minimum": 0,
+      "maximum": 9007199254740991
+    },
+    "ended": {
+      "type": "boolean",
+      "description": "If playback of the media has ended"
+    },
+    "loop": {
+      "type": "boolean",
+      "description": "If the video should restart after ending"
+    },
+    "muted": {
+      "type": "boolean",
+      "description": "If the media element is muted"
+    },
+    "paused": {
+      "type": "boolean",
+      "description": "If the media element is paused"
+    },
+    "percentProgress": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "The percent of the way through the media",
+      "minimum": 0,
+      "maximum": 100
+    },
+    "playbackRate": {
+      "type": "number",
+      "description": "Playback rate (1 is normal)",
+      "minimum": -9007199254740991,
+      "maximum": 9007199254740991
+    },
```

Comment:
Just thinking out loud, would an event with a `playbackRate` of `9007199254740991` (or `-9007199254740991`) actually be valid in any conceivable scenario?

Reply:
Not really... but there's nothing in your browser from stopping you from trying, so it's technical a valid value. It's pretty wild that negative values are even valid, what would one expect to happen... the video to play backwards?

https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/playbackRate

Reply:
Just to explain where I'm coming from on this, I'm thinking of programming mistakes where the rate is e.g. 125 instead of 1.25, leading to (what is in fact) bad events ending up in atomic tables. But I can also see how you're basically trying to be aligned with what the browsers allow so I'll resolve this.